src/view/com/posts/FeedItem.tsx

```diff
@@ -196,6 +196,12 @@ let FeedItemInner = ({
     },
   ]
 
+  let blockedParentAuthor: AppBskyActorDefs.ProfileViewBasic = {
+    did: '',
+    displayName: '[user blocked you]',
```

Comment:
If I block this user instead of them blocking me, this text might be confusing. Should probably do something more generic like `[blocked user]` and then checking for all forms of blocks on the viewer state when displaying it:

```
  blockedBy?: boolean
  blocking?: string
  blockingByList?: AppBskyGraphDefs.ListViewBasic
```

Reply:
I'm home in 30m and will adjust this then :)